# This file is auto-generated by the dh-umc tools and should not be edited!
msgid ""
msgstr ""
"Project-Id-Version: ucs-school-umc-computerroom\n"
"Report-Msgid-Bugs-To: packages@univention.de\n"
"POT-Creation-Date: 2013-05-08 05:31-0400\n"
"PO-Revision-Date: \n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: umc/python/computerroom/italc2.py:581
msgid "A HTTP error occured while contacting the OPSI server."
msgstr "Ein HTTP-Fehler beim kontaktieren des OPSI-Servers ist aufgetreten."

#: umc/python/computerroom/__init__.py:327
msgid "A different user is already running a computerroom session."
msgstr "Ein anderer Benutzer hat bereits eine offene Computerraumsitzung."

#: umc/python/computerroom/__init__.py:180
msgid "Action unavailable for UCC clients."
msgstr "Aktion nicht verf端gbar f端r UCC-Klienten."

#: umc/python/computerroom/__init__.py:82
#, python-format
msgid "Invalid room DN: %s"
msgstr "Ung端ltige Raum-DN: %s"

#: umc/python/computerroom/italc2.py:589
msgid ""
"The computer could not be reset due to an unknown error. Further information "
"are available in the logfiles."
msgstr ""
"Der Computer konnte augrund eines unbekannten Fehlers nicht zur端ckgesetzt "
"werden. Weitere Informationen befinden sich in den Protokolldateien."

#: umc/python/computerroom/italc2.py:587
msgid "The credentials for the OPSI server are incorrect."
msgstr "Die Zugangsdaten des OPSI-Servers sind falsch."

#: umc/python/computerroom/italc2.py:560
msgid "The credentials for the OPSI server could not be determined."
msgstr "Die Zugangsdaten des OPSI-Servers konnten nicht ermittelt werden."
